web2/src/pages/settings/PlexSettingsPage.tsx

```diff
@@ -219,22 +259,221 @@ export default function PlexSettingsPage() {
             </FormControl>
           </Grid>
         </Grid>
-        <Box>
-          <FormControl></FormControl>
-          <FormControl>
-            <TextField defaultValue={streamSettings.audioCodecs} />
-          </FormControl>
-          <FormControl></FormControl>
-        </Box>
-      </Paper>
+      </>
+    );
+  };
+
+  const renderAudioSettings = () => {
+    if (streamSettingsPending) {
+      return (
+        <Paper sx={{ display: 'flex' }}>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+        </Paper>
+      );
+    }
+
+    if (!streamSettings) {
+      return <div>Error</div>;
+    }
+
+    return (
+      <>
+        <Typography component="h6" sx={{ my: 2 }}>
+          Audio Options
+        </Typography>
+        <Grid flex="1 0 50%" container spacing={3}>
+          <Grid item xs={12} sm={6}>
+            <FormControl fullWidth>
+              <TextField
+                id="component-outlined"
+                label="Audio Codecs"
+                defaultValue={streamSettings?.audioCodecs}
```

Comment:
we're gonna have to add some defaults here so we dont get warnings about uncontrolled => controlled inputs from React when going from undefined => value